dotcom-rendering/src/amp/components/Footer.tsx

```diff
@@ -183,24 +184,27 @@ const FooterLinks: React.FC<{
 	);
 };
 
-export const Footer: React.FC<{ nav: NavType }> = ({ nav }) => (
-	<footer css={footer}>
-		<div css={innerContainerStyles}>
-			<div css={footerInner}>
-				<FooterLinks links={footerLinksNew} nav={nav} />
+export const Footer: React.FC<{ nav: NavType }> = ({ nav }) => {
+	const { group } = useContentABTestGroup();
+	return (
+		<footer data-content-test-group={group} css={footer}>
```

Comment:
Here we include a `data-content-test-group` attribute so we can inspect a given AMP article to see the assigned group.